sdm_versions_manager/database.py

```diff
@@ -0,0 +1,80 @@
+# Contains functions to connect to MongoDB, create the database, and perform CRUD operations.
+
+from pymongo import MongoClient
+import os
+
+
+# Load MongoDB connection details from environment variables or configuration
+MONGO_URI = os.getenv('MONGO_URI')
+DB_NAME = os.getenv('DB_NAME')
+COLLECTION_NAME = os.getenv('COLLECTION_NAME')
+
+
+def document_exists(data):
+    """Check if a document exists in the database based on subject, dataModel, and version.
+
+    Args:
+        data (dict): A dictionary containing the document data to check.
+
+    Returns:
+        bool: True if the document exists, False otherwise.
+    """
+    client = MongoClient(MONGO_URI)
```

Comment:
Is there no possible exceptions in this call?

Maybe this is a Tech. Debt to add in the list of issues for the moment...